node_modules/oae-core/linkpreview/js/linkpreview.js

```diff
@@ -135,8 +137,14 @@ define(['jquery', 'oae.core', 'jquery.oembed'], function($, oae) {
                     embeddable = false;
                 }
             }
+            // Only embed text, html, images, and types that the browser has plugins to support
+            var type = widgetData.previews.targetType? widgetData.previews.targetType.split(';')[0]: false;
+            var universalSupport = _.contains(['text/html', 'text/plain', 'image/png', 'image/jpeg', 'image/gif', 'image/svg+xml'], type);
+            var havePlugin = navigator.mimeTypes[type];
+            var supported = widgetData.previews.targetType && (universalSupport || havePlugin);
+
             // The link can be embedded as an iFrame
-            if (embeddable) {
+            if (embeddable && supported) {
```

Comment:
I'm not being prompted to download that link. Could you open your js console and run `navigator.mimeTypes['application/zip']` and verify that it returns `undefined`